manager/src/main/java/com/redhat/service/bridge/manager/RhoasServiceImpl.java

```diff
@@ -22,6 +22,16 @@
 
     @ConfigProperty(name = "rhoas.timeout-seconds")
     int rhoasTimeout;
+
+    @ConfigProperty(name = "rhoas.max_retries")
```

Comment:
I know it's too late but to be consistent with property naming so far this should be `rhoas.max-retries`